spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -78,24 +78,35 @@ Then to import the apps in bulk, use the `app import` command and provide the lo
 dataflow:>app import --uri file:///<YOUR_FILE_LOCATION>/stream-apps.properties
 ```
 
-For convenience, we have the static files with application-URIs (for both maven and docker) available for all the out-of-the-box
-Stream app-starters. You can point to this file and import all the application-URIs in bulk. Otherwise, as explained in
-previous paragraphs, you can register them individually or have your own custom property file with only the required application-URIs
-in it. It is recommended, however, to have a "focused" list of desired application-URIs in a custom property file.
-
-
-List of available static property files:
-
-* Maven based Stream Applications with RabbitMQ Binder: http://bit.ly/stream-applications-rabbit-maven
-* Maven based Stream Applications with Kafka Binder: http://bit.ly/stream-applications-kafka-maven
-* Docker based Stream Applications with RabbitMQ Binder: http://bit.ly/stream-applications-rabbit-docker
-* Docker based Stream Applications with Kafka Binder: http://bit.ly/stream-applications-kafka-docker
+For convenience, we have the static files with application-URIs (for both maven and docker) available 
+for all the out-of-the-box stream and task/batch app-starters. You can point to this file and import
+all the application-URIs in bulk. Otherwise, as explained in previous paragraphs, you can register them individually or have your own custom property file with only the required application-URIs in it. It is recommended, however, to have a "focused" list of desired application-URIs in a custom property file.
+
+List of available Stream Application Starters:
+
+[width="100%",frame="topbot",options="header"]
+|======================
+|Artifact Type |Stable Release |SNAPSHOT Release
+|RabbitMQ + Maven |link:http://bit.ly/1-0-4-GA-stream-applications-rabbit-maven[http://bit.ly/1-0-4-GA-stream-applications-rabbit-maven] |link:http://bit.ly/1-1-0-SNAPSHOT-stream-applications-rabbit-maven[http://bit.ly/1-1-0-SNAPSHOT-stream-applications-rabbit-maven]
+|RabbitMQ + Docker|link:http://bit.ly/1-0-4-GA-stream-applications-rabbit-docker[http://bit.ly/1-0-4-GA-stream-applications-rabbit-docker] | NA
```

Comment:
Why aren't there "SNAPSHOT" docker versions? We should have a file with the `latest` tag (bot in it and in its name IMO)